src/utils.js

```diff
@@ -107,19 +107,19 @@ function parseMarkdownWithFrontMatter(string) {
     let frontMatterTypes = [
         {
             type: 'yaml',
-            startDelimiter: '---\n',
+            startDelimiter: '---',
             endDelimiter: '\n---',
             parse: (string) => yaml.safeLoad(string, {schema: yaml.JSON_SCHEMA})
         },
         {
             type: 'toml',
-            startDelimiter: '+++\n',
+            startDelimiter: '+++',
             endDelimiter: '\n+++',
             parse: (string) => toml.parse(string)
         },
         {
             type: 'json',
-            startDelimiter: '{\n',
+            startDelimiter: '{',
```

Comment:
Having new-line delimiter is important part as it validates the this is indeed yaml or toml.
For example, without new-line, following frontmatter is illegal:
```
---title: welcome---
```
It must be:
```
---
title: welcome
---
```

I suggest using more generic approach to identify new-lines that will be recognized by underlying operating system:

```js
const os = require('os');

startDelimiter: `+++${os.EOL}`
endDelimiter: `${os.EOL}+++`
```

Reply:
Same for file-paths, there is `path.sep` that is different between posix and windows environments.

But for url paths it should always be `/`, all urls are standard under any OS :)